istioctl/pkg/tag/templates/validatingwebhook.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: istiod-default-validator
+  labels:
+    app: istiod
+    istio: istiod
+    istio.io/rev: {{ or .Revision "default" }}
+    istio.io/tag: {{ .Tag }}
+    # Required to make sure this resource is removed
+    # when purging Istio resources
+    operator.istio.io/component: Pilot
+webhooks:
+  - name: validation.istio.io
+    clientConfig:
+      {{ if .URL -}}
+      url: {{ .URL }}
+      {{- else -}}
+      service:
+        {{ if .Revision -}}
+        name: istiod-{{ .Revision }}
+        {{- else }}
+        name: istiod
+        {{- end }}
+        namespace: {{ or .IstioNamespace "istio-system" }}
+        path: "/validate"
+      {{- end }}
+      caBundle: {{ or .CABundle "" }}
+    rules:
+      - operations:
+          - CREATE
+          - UPDATE
+        apiGroups:
+          - security.istio.io
```

Comment:
This list is not complete, and i see its not in manifests/ either. So that is a bug in current master branch I think

Reply:
Yup I noticed that was missing telem API group.